src/index.js

```diff
@@ -1,32 +1,214 @@
 import './style.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import '@fortawesome/fontawesome-free/js/all.js';
+import List from './modules/List.js';
+/* eslint-disable no-plusplus */
+const container = document.querySelector('.container');
+const form = document.querySelector('.form');
+const inputtxt = document.querySelector('#inp');
+const clearAll = document.querySelector('.btn');
 
-const tasks = [{
-  description: 'wash the dishes',
-  completed: false,
-  index: 1,
-},
-{
-  description: 'walk the dog',
-  completed: false,
-  index: 2,
-},
-];
+let itemArray = [] || JSON.parse(localStorage.getItem('items'));
 
-const container = document.querySelector('.container');
-tasks.forEach((task) => {
-  const list = document.createElement('li');
-  list.classList.add('thelist');
-  const input = document.createElement('input');
-  input.type = 'checkbox';
-  input.classList.add('check');
-  const vertical = document.createElement('div');
-  vertical.innerHTML = ' <i class="fa-solid fa-ellipsis-vertical"></i>';
-  vertical.classList.add('elipsis');
-  list.innerHTML = task.description;
-  list.style.listStyleType = 'none';
-  list.appendChild(vertical);
-  list.appendChild(input);
-  container.appendChild(list);
+const completeItem = () => {
+  const localData = localStorage.getItem('items');
+  const parsedData = JSON.parse(localData);
+  const eachItem = document.querySelectorAll('.span');
+  for (let i = 0; i < eachItem.length; i += 1) {
+    if (eachItem[i].classList.contains('strike')) {
+      parsedData[i].completed = true;
+    } else {
+      parsedData[i].completed = false;
+    }
+    localStorage.setItem('items', JSON.stringify(parsedData));
+  }
+};
```

Comment:
- [ ] Good job so far 👏 From the project requirements, the following function concerning `completed` status should be in a separate module. I think you could have put it in the `List.js` file you have in your modules folder. This will help you implement ES6 Modular JS features which help to keep your `index.js` thin. Please put the following function in the module (recommended List.js) then import it here 😉